brave/brave-core
# Speedreader stylesheet from component updater

Resolves https://github.com/brave/brave-browser/issues/11004

Partly addresses https://github.com/brave/brave-browser/issues/9652

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

For initial verification that the approach works, need to:

- Start the browser, enable SpeedReader, verify it works
- Update the stylesheet, publish updated via crx pacakger
- Trigger the component update via brave://components, verify version number has changed
- Verify styling of rewritten content has changed

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
